packages/core/src/utils/index.js

```diff
@@ -0,0 +1,5 @@
+import * as constants from './constants';
+import * as helpers from './helper-functions';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Explicitly import the specific member needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS2208)